decidim-accountability/spec/controllers/decidim/accountability/admin/projects_import_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Accountability
+    module Admin
+      describe ProjectsImportController, type: :controller do
+        routes { Decidim::Accountability::AdminEngine.routes }
+        describe "GET the import result process new" do
```

Comment:
In controller specs we generally describe the action that we are requesting here, i.e.
```suggestion
        describe "GET show" do
```

I know the convention is not followed everywhere throughout the core but with new code we should follow such conventions.